4차/류동준_다트_게임.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+def solution(dartResult):
+    dartResult = dartResult.replace('10','a')
+    p = re.compile('[\d\w\*\#]')
```

Comment:
```suggestion
    p = re.compile('\d+|[SDT\*\#]')
```

이 부분은 정규식을 `\d+|[SDT\*\#]`으로 변경하면 10을 a로 바꾸지 않아도 될 것 같아요

Reply:
오 | 좀쓸줄아는놈인가?